RLBotFramework/setup_manager.py

```diff
@@ -113,9 +117,26 @@ def run(self):
             except queue.Empty:
                 pass
             except Exception as ex:
-                print(ex)
+                self.logger.error(ex)
                 pass
 
+        self.logger.info("Shutting Down")

```

Comment:
can we actually move this shutdown code to its own place in runner?

Mainly I think we should have a way to dynamically kill bots without killing the main thread.

Reply:
Done, and also removed a few things that proved unnecessary